bin/create_config.py

```diff
@@ -4,6 +4,6 @@
 
 def create_config(config_path=os.environ['YANGCATALOG_CONFIG_PATH']):
     config = configparser.ConfigParser()
-    config._interpolation = configparser.ExtendedInterpolation()
+    config._interpolation = configparser.ExtendedInterpolation() # pyright: ignore
```

Comment:
Use `parser = ConfigParser(interpolation=ExtendedInterpolation())` instead?

Reply:
Ah, yeah that looks like the proper way to do it.